ajax/libs/jquery.iframetracker/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "filename": "jquery.iframetracker.min.js",
+  "name": "jquery.iframetracker",
+  "version": "1.1.0",
+  "description": "jQuery Plugin to track click on iframes (like Google Adsense or Facebook Like button).",
+  "main": "jquery.iframetracker.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/vincepare/iframeTracker-jquery.git"
+  },
+  "keywords": [
+    "jquery-plugin",
+    "iframe",
+    "youtube",
+    "adsense",
+    "advertising",
+    "click"
+  ],
+  "author": "Vincent Paré",
+  "license": "Apache-2.0",
+  "homepage": "https://github.com/vincepare/iframeTracker-jquery",
+  "requiredFiles": [
+    "jquery.iframetracker.min.js"
```

Comment:
@LeaYeh ~
If the missing minified file has been pointed in "filename" field, it could not have to add in "requiredFiles" field.